src/hotspot/share/runtime/deoptimization.cpp

```diff
@@ -958,10 +958,10 @@ Deoptimization::DeoptAction Deoptimization::_unloaded_action
 template<typename CacheType>
 class BoxCacheBase : public CHeapObj<mtCompiler> {
 protected:
-  static InstanceKlass* find_cache_klass(Symbol* klass_name) {
+  static InstanceKlass* find_cache_klass(Thread* thread, Symbol* klass_name) {
     ResourceMark rm;
```

Comment:
Is worth adding rm(thread) in line 962?

Reply:
Yes, that's a good suggestion since we have thread.